backend/src/nodes/nodes/utility/round.py

```diff
@@ -0,0 +1,129 @@
+from __future__ import annotations
+
+import math
+from enum import Enum
+from typing import Union
+
+from ...groups import conditional_group
+from ...node_base import NodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import EnumInput, NumberInput
+from ...properties.outputs import NumberOutput
+from . import category as UtilityCategory
+
+
+class RoundOperation(Enum):
+    FLOOR = "Down"
+    CEILING = "Up"
+    ROUND = "Either way"
```

Comment:
"Either way" is a really weird way to refer to normal/half up rounding. Maybe we could find a better way to refer to it? Maybe "To Nearest"?

Reply:
I agree it's very weird.  How about "round up"/"round down"/"round"?

Reply:
Or just floor/ceiling/round.  The reason  I didn't do that to begin with is I'm not sure whether normal people know those terms, or if that's math/programmer jargon.